frontend/i18next.scanner/fr/translation.json

```
@@ -518,32 +518,32 @@
   "Uploading…": "Téléversement en cours…",
   "You cannot change this member role because there are no other space managers.": "Vous ne pouvez pas modifier ce rôle de membre car il n'y a pas d'autres responsables.",
   "Share a news...": "Partager une actualité…",
-  "Sunday": "__NOT_TRANSLATED__",
-  "Monday": "__NOT_TRANSLATED__",
-  "Tuesday": "__NOT_TRANSLATED__",
-  "Wednesday": "__NOT_TRANSLATED__",
-  "Thursday": "__NOT_TRANSLATED__",
-  "Friday": "__NOT_TRANSLATED__",
-  "Saturday": "__NOT_TRANSLATED__",
-  "Su": "__NOT_TRANSLATED__",
-  "Mo": "__NOT_TRANSLATED__",
-  "Tu": "__NOT_TRANSLATED__",
-  "We": "__NOT_TRANSLATED__",
-  "Th": "__NOT_TRANSLATED__",
-  "Fr": "__NOT_TRANSLATED__",
-  "Sa": "__NOT_TRANSLATED__",
-  "January": "__NOT_TRANSLATED__",
-  "February": "__NOT_TRANSLATED__",
-  "March": "__NOT_TRANSLATED__",
-  "April": "__NOT_TRANSLATED__",
-  "May": "__NOT_TRANSLATED__",
-  "June": "__NOT_TRANSLATED__",
-  "July": "__NOT_TRANSLATED__",
-  "August": "__NOT_TRANSLATED__",
-  "September": "__NOT_TRANSLATED__",
-  "October": "__NOT_TRANSLATED__",
-  "November": "__NOT_TRANSLATED__",
-  "December": "__NOT_TRANSLATED__",
-  "MM/dd/yyyy": "__NOT_TRANSLATED__",
+  "Sunday": "Dimanche",
+  "Monday": "Lundi",
+  "Tuesday": "Mardi",
+  "Wednesday": "Mercredi",
+  "Thursday": "Jeudi",
+  "Friday": "Vendredi",
+  "Saturday": "Samedi",
+  "Su": "D",
+  "Mo": "L",
+  "Tu": "Ma",
+  "We": "Me",
+  "Th": "J",
+  "Fr": "V",
+  "Sa": "S",
+  "January": "Janvier",
+  "February": "Février",
+  "March": "Mars",
+  "April": "Avril",
+  "May": "Mai",
+  "June": "Juin",
+  "July": "Juillet",
+  "August": "Août",
+  "September": "Septembre",
+  "October": "Octobre",
+  "November": "Novembre",
+  "December": "Decembre",
+  "MM/dd/yyyy": "jj/mm/aaaa",
```

Not sure about this one. Shouldn't "MM/dd/yyyy" be translated  "mm/jj/aaaa"? Or is the format adapted here?

---

and if this format is used to display a date, the original letters should probably be kept like this:

```suggestion
  "MM/dd/yyyy": "dd/MM/yyyy",
```